src/evm_transition_tool/transition_tool_methods/_filesystem.py

```diff
@@ -0,0 +1,40 @@
+import os
+import stat
+import json
+from json import dump
+from typing import Any, Dict
+
+
+def _dump_files_to_directory(output_path: str, files: Dict[str, Any]) -> None:
```

Comment:
This is only convention and is not enforced, but class methods that start with `_` are intended to be private to the class.

These functions aren't part of a class, so I'd remove the `_` prefix.

```suggestion
def dump_files_to_directory(output_path: str, files: Dict[str, Any]) -> None:
```